src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -197,6 +200,14 @@ def do_mutator_plugin():
           strategy.MUTATOR_PLUGIN_STRATEGY, default=MUTATOR_PLUGIN_PROBABILITY))
 
 
+def do_dataflow_tracing():
```

Comment:
Just a heads up, this will likely conflict with https://github.com/google/clusterfuzz/pull/578 (I wouldn't block on that change since it should be easy enough to resolve, just an FYI).